bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -439,8 +440,9 @@ def _forward(_model_output=model_output):
             # then compact new token phrases and probabilities into 1-D tensor
             topk_tensor = topk_token_phrases(last_logits, self.tokenizer, topk=topk)  # [batch_size, (topk + 1), max_len]
 
-            original_loss = self.get_loss_fct(_model_output.logits, tokens['input_ids'])
-            message = f'Loss: {original_loss:.2f}'
```

Comment:
Showing the original_loss server_side is generally useful feedback to know if the model is performing as expected. Would it be possible to bring this back in future, or was it causing some specific issue?

Reply:
Let me see if I can bring it back!